9-17-24 cve updates (#3961)
spectrocloud/librarium: docs/docs-content/security-bulletins/reports/cve-2024-38428.md

## Our Official Summary

> This is a critical severity vulnerability that affects any Wget version up to and including 1.24.5. Wget parses URIs in

Reviewer: ```suggestion
This is a critical severity vulnerability that affects any Wget version up to and including 1.24.5. `wget` parses URIs in
```

> a way that causes userinfo to be considered part of the host if it contains a semicolon. This means that the host part

Reviewer: ```suggestion
a way that causes user information to be considered part of the host if it contains a semicolon. This means that the host part
```

of the URI could be interpreted incorrectly and be abused by attackers that control the userinfo. The CVE is only

> exploitable when a vulnerable Wget version is used in specific conditions. Risk of this vulnerability getting exploited

Reviewer: ```suggestion
exploitable when a vulnerable `wget` version is used in specific conditions. Risk of this vulnerability getting exploited
```
